backend/app/controllers/spree/admin/taxons_controller.rb

```diff
@@ -83,6 +84,10 @@ def destroy
       def taxon_params
         params.require(:taxon).permit(permitted_taxon_attributes)
       end
+
+      def resource_not_found
+        super({ redirect_url: admin_taxonomies_path, flash_class: Taxon })
```

Comment:
You can remove the curly braces here. It doesn't make much sense to include them especially since we're using keyword args.